ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -147,6 +147,24 @@ ip_netmask_to_network()
 	BCAST_ADDR=$(($NET_NUM + $bcasthost))
 }
 
+# check if specified IP belongs to the subnet
+ip_from_subnet() {
+	MYIP="$1"
+	NET="$2"
+	MASK="$3"
+
+	ip_netmask_to_network "$NET" "$MASK"
+	ip_to_num "$MYIP"
+
+	if [ "$num" -gt "$NET_NUM" ] && [ "$num" -lt "$BCAST_ADDR" ]; then
```

Comment:
Please use double brackets for consistency in our scripts.

The whole if/else condition can be rewritten as: `[[ "$num" -gt "$NET_NUM" && "$num" -lt "$BCAST_ADDR" ]]` being the last line in the function (there is no need for the if/else statements). However I leave it up to you because the current (larger) if statement can improve readability.